Update example of individual certs

milohuman opened the pull request:
current wording is confusing for customers. They may thing you can provide single certs for blob, table, and/or queue storage which is NOT supported. You need a single cert for acs all-up (as covered in the mandatory certs table)

azurecla commented:
Hi __@rustd__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (pranavra). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

steved0x commented:
I made this fix to another pull request (https://github.com/Azure/azure-content-pr/pull/17318) so I am closing this one.

steved0x commented:
This change looks good.

steved0x commented:
#sign-off

covetocove committed:
Update example of individual certs

current wording is confusing for customers. They may thing you can provide single certs for blob, table, and/or queue storage which is NOT supported. You need a single cert for acs all-up (as covered in the mandatory certs table)
